### PR TITLE
docs: add latest content about GraphQL in APISIX

### PR DESCRIPTION
**Pull request by Chever-John**
### What this PR does / why we need it:

APISIX recently updated support for json and GET. This PR corresponds to this increase.
#6343 

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/router-radixtree.md [hunk 1]
```diff
@@ -289,6 +291,28 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
 }'
 ```
 
+We can verify GraphQL matches in the following two ways:
+
+1. 使用 GraphQL 查询字符串
```
Comment (leslie-tsang) [inline docs/en/latest/router-radixtree.md:296]: Translation required.

File: docs/en/latest/router-radixtree.md [hunk 2]
```diff
@@ -289,6 +291,28 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+        owner {
+            name
+        }
+        repo {
+            created
+        }
+    }'
+    ```
+    
+2. 使用 JSON 格式
```
Comment (leslie-tsang) [inline docs/en/latest/router-radixtree.md:310]: Translation required.

**Review by leslie-tsang — CHANGES_REQUESTED** [review]

File: docs/en/latest/router-radixtree.md (hunk 1 above)
Comment (Chever-John) [reply docs/en/latest/router-radixtree.md]: Oh, sry.
now it is all right.

File: docs/en/latest/router-radixtree.md (hunk 2 above)
Comment (Chever-John) [reply docs/en/latest/router-radixtree.md]: Sry, it is all right now.

File: docs/en/latest/router-radixtree.md [hunk 3]
```diff
@@ -289,6 +291,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+    $ curl -H 'content-type: application/graphql' -X POST http://127.0.0.1:9080/graphql -d '
+    query getRepo {
+        owner {
+            name
+        }
+        repo {
+            created
+        }
+    }'
+    ```
```
Comment (shuaijinchao) [inline docs/en/latest/router-radixtree.md:308]: Remove indentation, formatting needs to be consistent across the document.

File: docs/en/latest/router-radixtree.md [hunk 4]
```diff
@@ -289,6 +291,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+
+2. JSON format
+
+    ```shell
+    $ curl -H 'content-type: application/json' -X POST http://127.0.0.1:9080/graphql --data '{"query": "query getRepo { owner {name } repo {created}}"}'
+    ```
```
Comment (shuaijinchao) [inline docs/en/latest/router-radixtree.md:314]: ditto

File: docs/zh/latest/router-radixtree.md [hunk 5]
```diff
@@ -290,6 +292,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+    $ curl -H 'content-type: application/graphql' -X POST http://127.0.0.1:9080/graphql -d '
+    query getRepo {
+        owner {
+            name
+        }
+        repo {
+            created
+        }
+    }'
+    ```
```
Comment (shuaijinchao) [inline docs/zh/latest/router-radixtree.md:309]: ditto

File: docs/zh/latest/router-radixtree.md [hunk 6]
```diff
@@ -290,6 +292,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+
+2. 使用 JSON 格式
+
+    ```shell
+    $ curl -H 'content-type: application/json' -X POST http://127.0.0.1:9080/graphql --data '{"query": "query getRepo { owner {name } repo {created}}"}'
+    ```
```
Comment (shuaijinchao) [inline docs/zh/latest/router-radixtree.md:315]: ditto

File: docs/en/latest/router-radixtree.md (hunk 3 above)
Comment (Chever-John) [reply docs/en/latest/router-radixtree.md:308]: OK

File: docs/zh/latest/router-radixtree.md (hunk 6 above)
Comment (Chever-John) [reply docs/zh/latest/router-radixtree.md]: Solve it!

File: docs/en/latest/router-radixtree.md [hunk 7]
```diff
@@ -289,6 +291,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+        ["graphql_root_fields", "has", "owner"]
+    ],
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "39.97.63.215:4000": 1
+        }
+    }
+}'
+```
```
Comment (shuaijinchao) [inline docs/en/latest/router-radixtree.md:336]: Why directly reuse the routing configuration of POST? You can add GET to the `methods` of your route configuration to support both GET and POST requests.

Comment (Chever-John) [reply docs/en/latest/router-radixtree.md:336]: It can be done as you say, let me try it.

Comment (Chever-John) [reply docs/en/latest/router-radixtree.md]: Finished it! 
Thanks for a review @shuaijinchao

**Review by shuaijinchao — DISMISSED** [review]

File: docs/en/latest/router-radixtree.md [hunk 8]
```diff
@@ -289,6 +291,34 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+
+2. JSON format
+
+```shell
+$ curl -H 'content-type: application/json' -X POST http://127.0.0.1:9080/graphql --data '{"query": "query getRepo { owner {name } repo {created}}"}'
+```
+
+3. Try 'GET' request match
```
Comment (juzhiyuan) [inline docs/en/latest/router-radixtree.md:316]:
```suggestion
3. Try `GET` request match
```

Comment (Chever-John) [reply docs/en/latest/router-radixtree.md]: Thanks for your review!
It helps me a lot!

**Review by tzssangglass — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Review by shuaijinchao — DISMISSED** [review]

**Review by juzhiyuan — DISMISSED** [review]
LGTM. Please update your PR's Checklist (don't ignore it 🤗)

File: docs/zh/latest/router-radixtree.md [hunk 9]
```diff
@@ -290,6 +292,36 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
...
+$ curl -H 'content-type: application/json' -X POST \
+http://127.0.0.1:9080/graphql --data '{"query": "query getRepo { owner {name } repo {created}}"}'
+```
+
+3. 尝试 `GET` 请求
+
+```shell
+curl -H 'content-type: application/graphql' -X GET \
+"http://127.0.0.1:9080/graphql?query=query getRepo { owner {name } repo {created}}" -g
+```
```
Comment (tzssangglass) [inline docs/zh/latest/router-radixtree.md:323]: It is best to be consistent whether the shell adds $ or not.

Comment (Chever-John) [reply docs/zh/latest/router-radixtree.md:323]: You are right.

**Review by tzssangglass — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by shuaijinchao — APPROVED** [review]

**Pull request merged**